src/skin/js/firstRun.js

```diff
@@ -0,0 +1,19 @@
+(function($) {
+
+$(window).on("load", function () {
+
+  function setSeenComic() {
```

Comment:
Oh, one more thing, right now the user might scroll down without engaging `smoothScroll` at all (such as by using the scrollbar). We should make sure to call `setSeenComic` if the user scrolled far enough down, regardless of how they got there.